content/en/docs/started/installing-kubeflow.md

```diff
@@ -39,7 +39,15 @@ Packaged distributions are developed and supported by their respective maintaine
         <td>Kubeflow on AWS</td>
         <td>Amazon Web Services (AWS)</td>
         <td>Amazon Elastic Kubernetes Service (EKS)</td>
-        <td>1.2</td>
```

Comment:
I would just leave this at 1.3 because it's the latest version that has been released (and all the 1.4/1.5 docs are the same as 1.3), and then add an External Website link to our AWSLabs ReadMe.

Reply:
leaving external link empty for now, we want users to start from docs here. github has only part of deployment docs